source/includes/fundamentals/auth-specify.rst

```diff
@@ -0,0 +1,29 @@
+You can specify your authentication mechanism and credentials when connecting
+to MongoDB using either of the following:
+
+- a connection string
+- a ``MongoCredential`` factory method
+
+A **connection string** or **connection uri** provides instructions on how
```

Comment:
`provides instructions on how` is a bit imprecise, as a connection string is declarative, not imperative.  Perhaps instead just say `specifies how`

Reply:
Or something similar if you don't want to use the same verb twice in a row.

Reply:
Sounds good, thanks.